src/parser/core/SPELL_CATEGORY.ts

```diff
@@ -0,0 +1,74 @@
+import { t } from '@lingui/macro';
+
+enum SPELL_CATEGORY {
+  ROTATIONAL,
+  ROTATIONAL_AOE,
+  ITEMS,
+  COOLDOWNS,
+  DEFENSIVE,
+  SEMI_DEFENSIVE,
+  OTHERS,
+  UTILITY,
+  HEALER_DAMAGING_SPELL,
+  CONSUMABLE,
+  HIDDEN,
+}
+
+export function getSpellCategoryName(category: SPELL_CATEGORY) {
+  switch (category) {
+    case SPELL_CATEGORY.ROTATIONAL:
+      return t({
+        id: 'core.abilities.spellCategories.rotational',
+        message: `Rotational Spell`,
+      });
+    case SPELL_CATEGORY.ROTATIONAL_AOE:
+      return t({
+        id: 'core.abilities.spellCategories.rotationalAoe',
+        message: `Spell (AOE)`,
+      });
+    case SPELL_CATEGORY.ITEMS:
+      return t({
+        id: 'core.abilities.spellCategories.items',
+        message: `Item`,
+      });
+    case SPELL_CATEGORY.COOLDOWNS:
+      return t({
+        id: 'core.abilities.spellCategories.cooldowns',
+        message: `Cooldown`,
+      });
+    case SPELL_CATEGORY.DEFENSIVE:
+      return t({
+        id: 'core.abilities.spellCategories.defensive',
+        message: `Defensive Cooldown`,
+      });
+    case SPELL_CATEGORY.SEMI_DEFENSIVE:
+      return t({
+        id: 'core.abilities.spellCategories.semiDefensive',
+        message: `Offensive & Defensive Cooldown`,
+      });
+    case SPELL_CATEGORY.OTHERS:
+      return t({
+        id: 'core.abilities.spellCategories.others',
+        message: `Spell`,
+      });
+    case SPELL_CATEGORY.UTILITY:
+      return t({
+        id: 'core.abilities.spellCategories.utility',
+        message: `Utility`,
+      });
+    case SPELL_CATEGORY.HEALER_DAMAGING_SPELL:
+      return t({
+        id: 'core.abilities.spellCategories.healerDamagingSpell',
+        message: `Damaging Spell`,
+      });
+    case SPELL_CATEGORY.CONSUMABLE:
+      return t({
+        id: 'core.abilities.spellCategories.consumable',
+        message: `Consumable`,
+      });
+  }
+
+  return 'Unknown Spell Category';
```

Comment:
This could be `default` case in the switch. Also missing translation here.

Reply:
I'm personally against translating this as it is primairly a developers thing to notice when changing anything in this file. 
What's a maintainers perspective on return statement? Should this be in or outside the switch-statement?

Reply:
We shouldn't translate strings that are for devs only IMO. extra work for no value since nobody should ever see them.

As far as the return vs default, either is fine with me. `default` might be a little cleaner but i'm not going to fuss over that.

Reply:
also i did the same thing for reviewing as @sMteX so as long as snapshots still pass i think this is okay after deciding what you want to do for default vs return

Reply:
Changed to a default: return in favor of argument of cleaner. Thanks both of you! snapshots are pretty much the same. I had an error during development but fixed to code to be the same result again.